sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_directory_client.hpp

```diff
@@ -71,15 +71,15 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
      * @return ShareDirectoryClient A directory client that can be used to manage a share directory
      * resource.
      */
-    ShareDirectoryClient GetSubShareDirectoryClient(const std::string& subDirectoryName) const;
+    ShareDirectoryClient GetSubDirectoryClient(const std::string& subDirectoryName) const;
```

Comment:
Is `Sub` a word or just a prefix and make a whole word together with `directory`?

Reply:
This comment is fantastic, I learned something, and also LMAO during the investigation.

Reply:
Will be changed to `GetSubdirectoryClient` btw.